nearlib/dev.js

```diff
@@ -26,23 +27,23 @@ module.exports = {
             options.accountId = devAccountName;
             options.key = devKey;
         }
-        const keyStore = (options.deps && options.deps.keyStore) || new nearlib.BrowserLocalStorageKeystore();
-        const nodeUrl = options.nodeUrl || (await this.getConfig()).nodeUrl || 'http://localhost:3030';
+        this.deps = options.deps || {};
+        this.deps.keyStore = this.deps.keyStore || new nearlib.BrowserLocalStorageKeystore();
```

Comment:
let's not modify passed options objects as I don't think developers expect it. You can use `Object.assign({}, ...` to create new object and merge new property values.

Reply:
done